content/docs/reference/routes/from.mdx

```diff
@@ -16,21 +17,62 @@ import TabItem from '@theme/TabItem';
 
 ## Summary
 
-**From** is the externally accessible URL for the proxied request.
+The **From** route is the externally accessible URL for a proxied HTTP request.
 
 Specifying `tcp+https` for the scheme enables [TCP proxying](/docs/capabilities/tcp/) support for the route. You may map more than one port through the same hostname by specifying a different `:port` in the URL.
 
 ## How to configure
 
-The `URL` must contain a **scheme** and **hostname** and cannot contain a path.
+The From URL must contain a **scheme** and **hostname**. It can't contain a path.
+
+### Port mapping behavior
```

Comment:
I think we might want to stick with the phrase "port matching" here. According to wikipedia ["port mapping"](https://en.wikipedia.org/wiki/Port_mapping) has a meaning in the context of NAT, which is something unrelated.

Reply:
Done.